src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -110,414 +110,414 @@
 public class SimpleEvents {
 	static {
 		Skript.registerEvent("Can Build Check", SimpleEvent.class, BlockCanBuildEvent.class, "[block] can build check")
-				.description("Called when a player rightclicks on a block while holding a block or a placeable item. You can either cancel the event to prevent the block from being built, or uncancel it to allow it.",
```

Comment:
You need to resolve your tab deletion for this class, it's making the diff hard to read.